docker-compose.yml

```
@@ -0,0 +1,12 @@
+version: "3.7"
+services:
+  app:
+    build:
+      context: .
+      target: development
+    ports:
+      - "8080:8080"
+    env_file:
+      - .env
+    volumes:
+    - ./todo_app:/todo_app
```

Your Dockerfile's ENTRYPOINT uses /app so we need to mount it there so changes get picked up.

```suggestion
    - ./todo_app:/app/todo_app
```

---

Added